.storybook/webpack.config.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-param-reassign */
+const path = require('path');
+const dotenv = require('dotenv');
+const custom = require('../config/webpack/webpack.common');
+
+const env = dotenv.config({path: path.resolve(__dirname, '../.env.staging')}).parsed;
+
+module.exports = ({config}) => {
+    config.resolve.alias = {
+        'react-native-config': 'react-web-config',
+        'react-native$': 'react-native-web',
+        '@react-native-community/netinfo': './mock',
```

Comment:
Sorry if I missed this in a previous review, but can we just do this?

```suggestion
        '@react-native-community/netinfo': path.resolve(__dirname, '../__mocks__/@react-native-community/netinfo.js'),
```

Then you don't need to create `src/libs/NetInfo/mock.js`

Reply:
Won't really work because we are not running in `jest` and that mock depends on it. We could maybe not use the default mock tho with the jest.fn() lemme try and get back to ya.

Reply:
Alright this worked after getting rid of the jest dependency so I got rid of that other file :) Nice one!